### DISCUSSION TITLE
Computer frozen

### DISCUSSION THREAD

**Discussion by Nc2meu2** [RainbowMiner/RainbowMiner#2064] [Q&A]
Hello community,

Has anyone come across the issue that your computer freezes and gets the warning on the picture?
![image](https://user-images.githubusercontent.com/101607809/167614323-ae6e7926-3e74-463c-bf83-0dbd3f692225.jpeg)

I had to force quit Rainbowminer with task manager and restart it. 

Any suggestions?

**Comment by RainbowMiner** [comment]
I suspect, that the CcminerKlausT + Neoscrypt algo did overload your machine. Neoscrypt is pretty power hungry, CcminerKlausT can be pretty hard on small CPUs (on which kind of CPU is this system running?), thus possibly creating havoc. Force-quitting RainbowMiner will also force quit all subprocesses, so the CcminerKlausT would have been terminated, too.
If you really have a smaller CPU, I would recommend having an eye on the CPU load during mining.

I will add another parameter, that will allow you to lower the mining priority for one specific miner. At the moment, you can only lower the priority globally for all GPU (or CPU) miners. Also, you could try to lower it in config.txt and see, if the error stays away: `  "MiningPriorityGPU": "-2",` (that would lower the GPU mining priority down from the default -1).

If nothing helps - just disable the CcminerKlausT for now (add it to the exclude miner name list in config.txt: `"ExcludeMinerName": "CcminerKlausT",`).

So these are my follow-up questions:
- which CPU is working in your system?
- which Nvidia driver version is currently installed?

**Reply by Nc2meu2** [reply]
Thank you for the fast response and help,

The NVIDIA drivers are 512.59
CPU intel Celeron 1037u 1.80ghz. 

CPU is at 100% usage. Wow. 

My config file has :
"Miningprioritycpu":-2
"Miningprioritygpu":-1

I'm not sure if I understood correctly, a side from my GPUs, my cpu is being used for mining?
If that's the case, how do I stop that from happening. During the set up, I ensured that only my gpus were selected. 

I added ccminerklaust to excludeminername on config as well as added the mininprioritygpu to -2. Rebooted and hope for the best.

**Reply by RainbowMiner** [reply]
> I'm not sure if I understood correctly, a side from my GPUs, my cpu is being used for mining?

No, the CPU is not being used for mining. But some GPU miners (some more, some less) do process side calculations on the CPU, mostly quick checks on share validity. It's normally no problem, but the Celeron really is a bit on the low end. The -2 will most probably help.

**Reply by RainbowMiner** [reply]
Which pools are you using? Especially on a low end CPU rig, it's wise to have a smaller selection only (like 2-4 pools). Do not enable many many pools or the AutoCoin and AutoPool feature. This will keep the overhead even more in check.

**Reply by Nc2meu2** [reply]
During set up I chose 2miners, ethermine, minexmr, zergpool, zergpoolcoins, zpool. 
How do I disable auto pool and auto coins 

While checking the local host interface the all pools section shows me lots of algorithms for each pool, but I only see the pools I chose. 


Anything I should check?

**Reply by Nc2meu2** [reply]
Do I need to upgrade my CPU?
I just woke up to another unclean shot down. 

I turned it on and it just quit, closing all other applications.

**Reply by Nc2meu2** [reply]
I removed minexmr from the pool list and it hasn't cash today. Crossing fingers and hoping for the best.
